code/game/objects/items/weapons/holy_weapons.dm

```diff
@@ -283,7 +283,7 @@
 
 /obj/item/weapon/nullrod/whip
 	name = "holy whip"
-	desc = "What a terrible night to be on Space Station 13."
+	desc = "What a terrible night to be on the Cyberiad!"
```

Comment:
You could move the desc declaration to the New() proc for the dresser, since a variable expression like `[station_name()]` is evaluated at runtime and not at compile.